x-pack/plugins/enterprise_search/public/applications/shared/flash_messages/flash_messages_logic.ts

```diff
@@ -51,6 +58,15 @@ export const FlashMessagesLogic = kea<MakeLogicType<FlashMessagesValues, FlashMe
         clearQueuedMessages: () => [],
       },
     ],
+    toastMessages: [
+      [],
+      {
+        addToastMessage: (toasts, { newToast }) => [...toasts, newToast],
+        dismissToastMessage: (toasts, { removedToast }) =>
+          toasts.filter((toast) => toast.id !== removedToast.id),
+        clearToastMessages: () => [],
```

Comment:
This currently isn't being used anywhere, but I thought I should add it just in case we do need to clear toast messages in the future (e.g. on new page view, on form re-submit).

That being said I can remove it from the PR if folks prefer not to have it until we need it, just LMK!